CONTRIBUTING.md

```diff
@@ -57,25 +57,39 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://git.k8s.io/community/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Releases
+
+Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track releases.
+
+- Minor versions CAN be planned and scheduled twice / year.
+- Patch versions CAN be planned and scheduled each month.
+- Code freeze is in effect 72 hours (3 days) before a release.
+  - Only critical bug fixes may be merged in between freeze & release.
+    - Each bug MUST be associated with an open issue and properly triaged.
+    - PRs MUST be approved by at least 2 project maintainers.
```

Comment:
Currently only 1  `approve` is enough. Is this going to be enforced somewhere?

Reply:
Probably not, just going to be diligent about it with a `/hold`

Reply:
might be add something like
```
In order to achieve this, during code-freeze:
- the first approver should apply /approve followed by /hold
- the second approver should apply /approve followed by /hold cancel
```